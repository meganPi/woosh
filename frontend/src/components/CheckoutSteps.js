import React from 'react';

//Create checkout wizard at top of screen, so that user can track where they are in
//the checkout process.
function CheckoutSteps(props) {
  return <div className="checkout-steps">
    <div className={props.step1 ? 'active' : ''} >Signin</div>
    <div className={props.step2 ? 'active' : ''} >Shipping</div>
    <div className={props.step3 ? 'active' : ''} >Payment</div>
    <div className={props.step4 ? 'active' : ''} >Place Order</div>
  </div>
}

export default CheckoutSteps;