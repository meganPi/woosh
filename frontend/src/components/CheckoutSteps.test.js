import React from 'react';
import renderer from 'react-test-renderer';

import CheckoutSteps from './CheckoutSteps';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<CheckoutSteps />).toJSON();
  expect(tree).toMatchSnapshot();
});