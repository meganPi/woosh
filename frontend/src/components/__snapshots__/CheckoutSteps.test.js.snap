// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="checkout-steps"
>
  <div
    className=""
  >
    Signin
  </div>
  <div
    className=""
  >
    Shipping
  </div>
  <div
    className=""
  >
    Payment
  </div>
  <div
    className=""
  >
    Place Order
  </div>
</div>
`;
